src/Psalm/Internal/Analyzer/Statements/Expression/Call/ArgumentAnalyzer.php

```diff
@@ -940,6 +940,7 @@ public static function verifyType(
                         || $context->inside_use
                         || $context->inside_assignment
                         || $context->inside_conditional
+                        || $context->inside_throw
```

Comment:
I'm not sure whether I really needed to add `$context->inside_throw` to _all_ of these places, but no tests broke :man_shrugging: